blueflood-http/src/main/java/com/rackspacecloud/blueflood/http/DefaultHandler.java

```diff
@@ -33,7 +36,18 @@ public void handle(ChannelHandlerContext ctx, HttpRequest request) {
     }
 
     public static void sendResponse(ChannelHandlerContext channel, HttpRequest request, String messageBody, HttpResponseStatus status) {
+        sendResponse(channel, request, messageBody, null);
+    }
```

Comment:
NIT:  spacing (need new line).  But wait and see if there's other changes you'll need to make from the other reviewers.